packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -48,14 +47,27 @@ export const DefaultCell = (props: TableCellProps) => {
     }
   }
 
+  const isStringValue = typeof value === 'string';
+
+  const cellStyle = getCellStyle(tableStyles, cellOptions, displayValue, inspectEnabled, isStringValue);
+
+  if (isStringValue) {
+    if (cellProps.style?.justifyContent === 'flex-end') {
```

Comment:
so this a bit wonky because the current API allows passing in arbitrary style, and we see thee styles being used with the implied assumption that they're gonna be applied to a contained that's `display: flex`, which is kinda leaky. this api can be improved to specify cell alignment explicitly rather than relying on passing in raw css props.